L1Trigger/L1TCaloLayer1/src/L1TCaloLayer1FetchLUTs.cc

```diff
@@ -235,12 +254,12 @@ bool L1TCaloLayer1FetchLUTs(const edm::EventSetup& iSetup,
         auto index = std::distance(hfScalePhiBins.begin(),pos);
         if (index<18) {
           caloEta*=-1;
-          iPhi = index*4+1;
+          iPhi = index*4-1;
         }
         else {
-          iPhi = (index-18)*4+1;
+          iPhi = (index-18)*4-1;
         }
-        if (abs(caloEta) == 41 && iPhi<3) iPhi = 3;
```

Comment:
ieta 40 and 41 both only have valid towers if `ieta % 4 in [3,4]`
This logic should guarantee that fact